index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="resume">
+        <h1 class="resume__text">Resume</h1>
+      </div>
+
+      <div class="header photo">
+        <img
+          class="photo__img"
+          src="./img/MyPhoto.png"
+          alt="MyPhoto"
+          width="300px"
+          height="300px"
+        />
+      </div>
+      <div class="header info">
+        <div class="info wrap">
+          <div class="info__text">Prushak</div>
+          <div class="info__text">Evgen</div>
+          <div class="info__text">Vladimirovish</div>
+          <div class="info__text">Javascript Developer</div>
+        </div>
+      </div>
+    </header>
+
+    <section class="scills">
+      <h3 class="scills__text">Objective & Summary</h3>
+      <ul class="scills list">
+        <li class="list__item">
+          Start a career in EPAM and in 3 years to grow to team lead
+        </li>
+      </ul>
+
+      <h3 class="scills__text">Key skills:</h3>
+
+      <ul class="scills list">
+        <li class="list__item">
+          Programming languages: Java Script – basics
+        </li>
+        <li class="list__item">Development Tools: Visual Studio Code</li>
+        <li class="list__item">
+          Web Technologies: HTML\СSS\Java Script – basics
+        </li>
+        <li>Defect & bug discovery</li>
+      </ul>
+

```

Comment:
Why do you use some li in one line and for another you use line breaks? it's difficult to review the code above